target/linux/ar71xx/image/senao.mk

```diff
@@ -0,0 +1,28 @@
+define Build/senao-factory-image
+	$(eval board=$(word 1,$(1)))
+	$(eval rootfs=$(word 2,$(1)))
+
+	mkdir -p $@.senao
+
+	touch $@.senao/FWINFO-OpenWrt-$(REVISION)-$(board)
+	$(CP) $(IMAGE_KERNEL) $@.senao/openwrt-senao-$(board)-uImage-lzma.bin
+	$(CP) $(rootfs) $@.senao/openwrt-senao-$(board)-root.squashfs
+
+	$(TAR) -czf $@ -C $@.senao .
+	rm -rf $@.senao
+endef
+
+define Device/ens202ext
+  DEVICE_TITLE := EnGenius ENS202EXT
+  BOARDNAME := ENS202EXT
+  DEVICE_PACKAGES += rssileds
+  CONSOLE := ttyS0,115200
```

Comment:
This is a default value, defined in `target/linux/ar71xx/image/Makefile` (take a look please at `define Device/Default`), you can drop it.